1-exercises/A-accessing-values/exercise3.js

```diff
@@ -19,7 +19,7 @@ let basketballTeam = {
     - sorts the top players in alphabetical order
     - console.logs the name of each player on a new line
 */
-
+console.log(basketballTeam.topPlayers.sort().toString())
 // write code here
 
```

Comment:
great job :), I think here we wanted you to show each top player, instead of showing the string, at the moment your result is like this : `"Dennis Rodman,Michael Jordan,Scottie Pippen"`  but we wanted you to show the player names like this : 
`basketballTeam.topPlayers.sort().forEach/map(player => console.log(player))`